### ISSUE TITLE
-c flag doesn't actually force color when piped (iow: when output not TTY)

### ISSUE THREAD

**Issue by hraban** [pinojs/pino-pretty#231]
## Reproduce

```
$ echo '{"message": "Foo", "level":"warn"}' | npx pino-pretty -c | xxd
```

## Expected output

```
00000000: 1b5b 3333 6d57 4152 4e1b 5b33 396d 3a0a  .[33mWARN.[39m:.
00000010: 2020 2020 6d65 7373 6167 653a 2022 466f      message: "Fo
00000020: 6f22 0a                                  o".
```

Notice the `.[33`, or `1b5b3333` escape before `WARN`: that's color escape.

## Actual output

```
00000000: 5741 524e 3a0a 2020 2020 6d65 7373 6167  WARN:.    messag
00000010: 653a 2022 466f 6f22 0a                   e: "Foo".
```

plain output, no colors.

## Further comments

You can currently force colors by passing a colorette specific envvar:

```
$ echo '{"message": "Foo", "level":"warn"}' | FORCE_COLOR=1 npx pino-pretty | xxd
```

Note: no `-c` needed.

If this is the de facto official pino-pretty API, then I'd vote for documenting that explicitly, and removing the `-c` option altogether.

It would, however, tie you in to this API regardless of what lib you use, so if you ever move away from colorette, you're still tied to this.

**Comment by mcollina** [comment]
What version are you using? This should be fixed right now in v7.0.1.

**Comment by hraban** [comment]
what a coincidence. :) yeah I had npm installed it when 7.0.0 was the latest, and by the time the ticket was written, 7.0.1 was released.
